content/departments/engineering/teams/source/index.md

```diff
@@ -18,7 +19,7 @@ For more details, read our [Source strategy & mission](strategy.md)
 
 ## Contact
 
-- #ask-source channel or @source-support in Slack.
+- #discuss-source in Slack.
```

Comment:
```suggestion
- #discuss-source or @source-support in Slack.
```

I think we can keep that, this is the handle people are supposed to tag instead of individuals where possible :)

Reply:
Ah good catch thanks